.travis.yml

```diff
@@ -5,6 +5,10 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.4
```

Comment:
I've long since disabled travis on my account. Any chance you have some time to setup tests in github actions?

Reply:
Yes, I'm doing that right now as well.